app/assets/javascripts/components/Newsletter.js

```diff
@@ -0,0 +1,72 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Newsletter,
+      defaultConfig = {};
+
+  Newsletter = function($el, config) {
+    Newsletter.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(Newsletter);
+
+  /**
+  * Set up and populate the model from the form inputs
+  * @param {Promise} initialised
+  */
+  Newsletter.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this.showInArticle($);
+  };
+
+  Newsletter.prototype.showInArticle = function($) {
+    var $panel = this.getPanel($);
+    var $article = $('.editorial main');
+
+    if (!$panel.length || !$article.length) {
+      setTimeout(function () {
+        this.showInArticle($);
```

Comment:
@stevoland - any idea why this bit was in the optimisely script? Not sure on a case example of why the panel or article would ever be missing at this point?